pysat/tests/test_utils_coords.py

```diff
@@ -60,379 +36,83 @@ def test_adjust_cyclic_data_custom(self):
         assert test_angles.max() < 180.0
         assert test_angles.min() >= -180.0
 
+
+class TestLonSLT():
+    def setup(self):
+        """Runs after every method to clean up previous testing."""
+        self.test_inst = None
```

Comment:
is it worth moving away from testInst?

Reply:
We might want to move away from using test as part of the variable names in general.  I don't think this causes any specific issues here, but functions that begin with "test" will be collected as part of the unit tests regardless of where they are in the code. I'm kind of leaning toward reserving "test" as a keyword for functions and classes that are meant to be run as tests.

Reply:
I changed it to `py_inst` after discussion with @jklenzing